### PR TITLE
Bryan

### PR DESCRIPTION
**Pull request by Rayudo11**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,14 +3,21 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@syncfusion/ej2-react-dropdowns": "^20.1.51",
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.1.1",
     "@testing-library/user-event": "^13.5.0",
-    "axios": "^0.26.1",
```
Comment (MHargrove96) [inline package.json:9]: why did you delete Axios? Are you using another library that i am not seeing?

Comment (Rayudo11) [reply package.json:9]: my bad i hadn't downloaded yet. fixing it now.

File: src/index.js [hunk 2]
```diff
@@ -1,9 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+<<<<<<< HEAD
+import App from './SortByInput';
```
Comment (MHargrove96) [inline src/index.js]: hey i think you may have imported this wrong App i believe come from app.js

File: src/components/SearchBarInput.jsx [hunk 3]
```diff
@@ -17,4 +17,8 @@ function SearchBarInput() {
   )
 }
 
-export default SearchBarInput
\ No newline at end of file
+<<<<<<< HEAD
+export default SearchBarInput;
+=======
+export default SearchBarInput
```
Comment (MHargrove96) [inline src/components/SearchBarInput.jsx]: this is a component that i had built, did you change something that i am not seeing or what happened can you explain a bit?

**Review by MHargrove96 — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (Rayudo11) [reply src/index.js]: I got a bit confused. I'm trying to to push changes now.

**Comment by Rayudo11** [comment]
I pushed the new changes. It should be working properly now.

**Review by MHargrove96 — APPROVED** [review]
